### PR TITLE
ISPN-12325 Hot Rod server instances in tests use too much memory

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.redhat.com/browse/ISPN-12325

### CODE REVIEW

**Comment by danberindei** [comment]
@wburns `ITestResult.getInstance()` returns the actual test instance, and `ITestResult` instances are held forever so that they can be passed to the reporter at the end of the test run. IDK if TestNG could set the instance to `null` after the test class finishes or not.

File: server/runtime/src/main/ant/infinispan-server.xml [hunk 1]
```diff
@@ -87,30 +87,9 @@
         </sequential>
     </macrodef>
     <target name="-do.kinit" depends="-check.skipped" unless="tests.skipped">
-        <exec executable="bash" outputproperty="ps.pid" resultproperty="ps.result" failifexecutionfails="false"
```
Comment (wburns) [inline server/runtime/src/main/ant/infinispan-server.xml:90]: Did you mean to include these changes in this PR?

Comment (danberindei) [reply server/runtime/src/main/ant/infinispan-server.xml]: No, but the PR with those changes  (#8679) got integrated sometime ago, so I'm not sure why GitHub was still showing them. I pushed again and they disappeared from the diff view.

**Comment by wburns** [comment]
Integrated into master, thanks @danberindei !

**Pull request merged**